assets/src/edit-story/components/draganddrop/index.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { useUploader } from '../../app/uploader';
+import { ReactComponent as UploadIcon } from './icons/upload.svg';
+
+const DragandDropComponent = styled.div`
+  min-width: 100%;
+  min-height: 100%;
+`;
+const DragandDropOverContent = styled.div``;
+
+const DragandDropOverLayWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.6);
+  position: absolute;
+  top: 56px;
```

Comment:
If this is to offset for the tab bar, sync with Sam on whether it's actually better to cover the tab bar as well. It's not going to be clickable anyways as you're dragging and it looks a bit odd uncovered. Otherwise, let's have the dropzone wrap the content of the tab instead of the entire panel and get rid of the `56px`

Reply:
In the design I used to implement, the tab bar was visual. I personally done like this, so can change it back to `0` as it has changed in the design anyway.

Reply:
IMHO we should cover it all. But if they want to keep it showing, I believe we need to place the DND overlay in the right container.

Reply:
I.e., yes, it should be `0`.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/175/commits/cce35b1409450e194144e31f377a75f5247886d3